src/libpgagroal/management.c

```diff
@@ -1610,6 +1613,27 @@ write_ssl(SSL* ssl, void* buf, size_t size)
    return 1;
 }
 
+static int
+write_header_info(char* header, bool start, char* command) {
+   char header_info[42] = " {sender: ";
```

Comment:
I would remove the space after the colon, is it useful?

Reply:
Th spaces give a debug message like this one:
`2024-03-18 17:47:16 DEBUG management.c:1635 Header:  {sender: pgagroal version: 1.7.0}
`